src/Symfony/Component/Notifier/Bridge/Bandwidth/BandwidthTransport.php

```diff
@@ -60,31 +60,26 @@ protected function doSend(MessageInterface $message): SentMessage
         if (!$message instanceof SmsMessage) {
             throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
         }
-        $opts = $message->getOptions();
-        $options = $opts ? $opts->toArray() : [];
+        $options = $message->getOptions()?->toArray() ?? [];
         $options['text'] = $message->getSubject();
-        $options['from'] = $options['from'] ?? $this->from;
-        $options['to'] = $options['to'] ?? [$message->getPhone()];
-        $options['account_id'] = $options['account_id'] ?? $this->accountId;
-        $options['applicationId'] = $options['application_id'] ?? $this->applicationId;
-        unset($options['application_id']);
-
-        if (!isset($options['priority']) && $this->priority) {
-            $options['priority'] = $this->priority;
-        }
-
-        if (!preg_match('/^\+[1-9]\d{1,14}$/', $this->from)) {
-            throw new InvalidArgumentException(sprintf('The "From" number "%s" is not a valid phone number. The number must be in E.164 format.', $this->from));
+        $options['from'] = $message->getFrom() ?: $this->from;
+        $options['to'] = [$message->getPhone()];
+        $options['accountId'] ??= $this->accountId;
```

Comment:
Before we used account_id kn the request, now accountId, is it correct or did I overlooked sth?

Reply:
The option array should be aligned with the API. The API uses `accountId`, that's why.